.github/workflows/docker.yml

```diff
@@ -70,15 +70,16 @@ jobs:
         env:
           DATABASE_URL: ${{ steps.pg.outputs.connection-uri }}
 
-      - name: Login to DockerHub
+      - name: Login to GitHub Docker registry
         uses: docker/login-action@v2
         if: ${{ github.actor != 'dependabot[bot]' && !github.event.pull_request.head.repo.fork }}
         with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Push the Docker image
-        if: ${{ github.actor != 'dependabot[bot]' && github.event_name != 'pull_request' }}
+        if: ${{ github.actor != 'dependabot[bot]'}}
```

Comment:
why would you want to run this on a pull request?

Reply:
No, that's only for testing purposes.